elife-00666.xml

```diff
@@ -1797,7 +1797,7 @@ not add all rids to the xref as it does not work. Only add the first one -->
                                     <title>Title of the source code.</title>
                                     <p>Legend of the source code.</p>
                                 </caption>
-                                <media mimetype="application" mime-subtype="xml" xlink:href="elife-00666-fig4-code1.xlsx"/>
+                                <media mimetype="application" mime-subtype="xlsx" xlink:href="elife-00666-fig4-code1.xlsx"/>
```

Comment:
The change here is to make it match the `.xlsx` file extension. `application/xml` is a mime type (for XML documents), but `application/xlsx` isn't, it's `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`. `mime-subtype` isn't the same thing as the file extension.